src/integrationtests/java/com/aws/greengrass/integrationtests/BaseITCase.java

```diff
@@ -66,7 +66,6 @@ public class BaseITCase {
 
     @BeforeEach
     void setRootDir() {
-        System.setProperty("aws.greengrass.scanSelfClasspath", "true");
```

Comment:
Would this break other plugin integ tests? like log manager. I think they extend BaseITCase. https://github.com/aws-greengrass/aws-greengrass-log-manager/blob/main/src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java#L97

Reply:
Yes this is mentioned in the description. Plugins will need to set it for their integration tests. It speeds us up here so that's worthwhile imo